src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -3382,13 +3382,14 @@ private function check_for_page_caching() {
 			$headers['Authorization'] = 'Basic ' . base64_encode( wp_unslash( $_SERVER['PHP_AUTH_USER'] ) . ':' . wp_unslash( $_SERVER['PHP_AUTH_PW'] ) );
 		}
 
+		$caching_url                   = apply_filters( 'site_status_persistent_page_cache_url', home_url( '/' ) );
```

Comment:
Filter doc missing. https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters

Reply:
Added description for filter. I'm not sure what version to specify in "since", specified the following "6.6.3"

Reply:
Added description for filter. I'm not sure what version to specify in "since", specified the following "6.6.3"

Reply:
@zetrider, let's use 6.7.